Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ruby:2.3.1
+MAINTAINER Kevin Reedy <kreedy@chef.io>
+
+RUN apt-get update && apt-get install -y fakeroot
+
+ADD . /opt/chefdk-build
+
+WORKDIR /opt/chefdk-build/omnibus
+RUN bundle install --without development
+RUN git config --global user.email "nobody@chef.io" && git config --global user.name "Chef"
+RUN bundle exec omnibus build chefdk && bundle exec omnibus clean chefdk
```

Comment:
An omnibus build is triggered for every commit. If the build succeeds it goes to the `unstable` channel which is only fetchable from behind the VPN. If the tests succeed it goes to the `current` channel. When we release we manually promote to the `stable` channel. You can specify `-c current` to the bash command I posted above to get the builds that are generated on every commit.

I think it is better to download and install than rebuild because we also run tests as part of the build process. But I'm 👍 on this PR either way you do it, and we can always address it later if something goes wrong.